### DISCUSSION TITLE
can support ipv6?

### DISCUSSION THREAD

**Discussion by wangxu1987** [lobehub/lobehub#1912] [Self Hosting] [answered]
The container is deployed to my NAS, but it does not support IPv6 external network access. How can I solve this problem?

**Comment by BrandonStudio** [comment, marked as answer]
How did you deploy it? Docker compose?
If so, you can either [enable IPv6 for docker](https://docs.docker.com/config/daemon/ipv6) or use a reverse proxy such as Nginx.
